analyses/cell-type-wilms-tumor-14/run_cell-type-wilms-14.sh

```diff
@@ -23,6 +23,8 @@ scratch_dir="scratch"
 results_dir="results"
 data_dir="../../data/current/SCPCP000014"
 
+ls -l ${data_dir}
```

Comment:
Did you want to keep this too, or just the echos?

Reply:
No, I can remove that
```suggestion
```